src/layer/vector/Polygon.js

```
@@ -84,6 +84,34 @@ L.Polygon = L.Polyline.extend({
 		return this._map.layerPointToLatLng(center);
 	},
 
+	// @method contains (latlng: LatLng): Boolean
+	// Returns `true` if `latlng` is inside the (multi)polygon.
```

So maybe also throw an error if `this._map` is undefined?